pkg/kubelet/network/kubenet/kubenet_linux.go

```diff
@@ -349,6 +350,11 @@ func (plugin *kubenetNetworkPlugin) TearDownPod(namespace string, name string, i
 		}
 	}
 	if err := plugin.delContainerFromNetwork(plugin.netConfig, network.DefaultInterfaceName, namespace, name, id); err != nil {
+		// This is to prevent returning error when TearDownPod is called twice on the same pod. This helps to reduce event pollution.
+		if !ok {
+			glog.Errorf("Failed to delete container which is not found in podCIDRs map: %v", err)
```

Comment:
nit: Add "from network plugin" to not confuse readers who may think kubelet failed to remove the container.